pkgs/development/python-modules/dataset/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, sqlalchemy
+, alembic
+, banal
+}:
+
+buildPythonPackage rec {
+  pname = "dataset";
+  version = "1.5.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0q7gvs5dnhv2hah784gx2kqxzna20apphbc9k5j9vmxvn4k34ghc";
+  };
+
+  propagatedBuildInputs = [
+    sqlalchemy alembic banal
+  ];
+
+  doCheck = false;
```

Comment:
Why are you disabling tests? A comment would be useful.
And if there's an error, can you maybe send it here?

Reply:
the tests seem to try to import a non-existent module named `test.test`

```
running build_ext
0:00:00 load avg: 0.19 Run tests sequentially
0:00:00 load avg: 0.19 [1/1] test
test test crashed -- Traceback (most recent call last):
  File "/nix/store/q6gfck5czr67090pwm53xrdyhpg6bx67-python3-3.8.9/lib/python3.8/test/libregrtest/runtest.py", line 270, in _runtest_inner
    refleak = _runtest_inner2(ns, test_name)
  File "/nix/store/q6gfck5czr67090pwm53xrdyhpg6bx67-python3-3.8.9/lib/python3.8/test/libregrtest/runtest.py", line 221, in _runtest_inner2
    the_module = importlib.import_module(abstest)
  File "/nix/store/q6gfck5czr67090pwm53xrdyhpg6bx67-python3-3.8.9/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 973, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'test.test'

test failed

== Tests result: FAILURE ==

1 test failed:
    test

Total duration: 72 ms
Tests result: FAILURE
builder for '/nix/store/ina80yb0m2kk0cnw097m7rmcsr5f3anx-python3.8-dataset-1.5.0.drv' failed with exit code 2
```

Reply:
Please add a comment with an explanation.